docs/about/release_notes.rst

```diff
@@ -27,6 +27,99 @@ Release Notes
  Miscellaneous:
    -
 
+
+Version 8.0
+-----------
+
+In this version of ERT we have clean up a lot of unused analysis keywords. Some have been removed,
+while others that were previously deprecated have now been removed. If they are present in your
+configuration, you will get a message in the suggestor at startup:
+
+.. image:: version-8.0-suggestor.png
+
+We encourage everyone to keep their configuration updated and remove and replace deprecated keywords.
+
+Another big change this release is the change to a streaming algorithm. The streaming algorithm
+will significantly reduce the memory usage when doing updates, without any effect to the result of the
+update.
+
+We have also added adaptive localisation for Ensemble smoother and ES-MDA, this can be activated by opening
+the analysis setting and checking the `Adaptive localization` check box. The correlation threshold
+can then be set, though a sensible default will be determined based on ensemble size. At a correlation threshold
+of 0.0 the update will be identical to no localization, while at a correlation threshold of 1.0 the posterior
+will be identical to the prior, and there will be no update. Note that the feature is still experimental, and
+any feedback is welcomed.
+
+.. image:: version-8.0-localization.gif
+
+We added an option to delete run path if it exists. Running in an existing run path can cause unintended side effects.
+For example ERT expects to read a number of files after the forward model has completed. If those files exists from a previous
+run, and for some reason the new forward model did not overwrite them, ERT will read outdated information. Example:
+
+.. image:: version-8.0-delete-runpath.gif
+
+Note that if the run path is large, deleting it might take several minutes.
+
+There is a new tab in the `Details` view of the `Run simulations` panel, showing the progress of the update. While in most cases the
+update is fairly quick, if running with Adaptive localization, the run time will be longer, and this tab will show the progress.
+
+.. image:: version-8.0-update.gif
+
+Full change log below:
+
+Breaking changes:
+  - Make random seed be int only (`link <https://github.com/equinor/ert/pull/6390>`__)
+  - Add events and a GUI model for base_run_model (`link <https://github.com/equinor/ert/pull/6388>`__)
+  - Store observations (`link <https://github.com/equinor/ert/pull/6302>`__)
```

Comment:
New feature maybe?

Reply:
Yhea. that I think deserves a separate point. But we have no logging indicating that this is done for anything else than the workflows we control.